Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.7
+
+WORKDIR /home/app
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
But isn't that the purpose of the `.dockerignore` file?

Reply:
Not really, `.dockerignore` only helps us ignore most known things but using `COPY . .` is still not a good practice since as you can copy confidential or unnecessary files into the container that might not be listed in `.dockerignore`. Here's a good article that Byron found talks about that and other good practices 🙂 

https://sysdig.com/blog/dockerfile-best-practices/#1-3